data/remnant/remnant ships.txt

```diff
@@ -655,6 +711,28 @@ ship "Merganser"
 	description "	The cloaking technology is built into the Merganser's hull; it cannot be removed or transferred to another ship."
 
 
+ship "Merganser" "Merganser II"
```

Comment:
```suggestion
ship "Merganser" "Merganser (II)"
```
This should potentially be `Mark II` for consistency with the Navy ships and I think maybe some others.

Reply:
I consider "Mark II" to be a particular naming convention for the Navy. In this particular case the (II) is appropriate, though, as this is merely a different configuration for the Merganser, it isn't actually a different ship.